### DISCUSSION TITLE
Common problem of connection refused and two services are not able to communicate between them because of rpc or Grpc

### DISCUSSION THREAD

**Discussion by Sagar9921** [seaweedfs/seaweedfs#6309] [General]
What is Grpc and rpc and how to overcome problems related to this?

error : 
I1202 06:47:37.826718 volume_grpc_client_to_master.go:43 checkWithMaster localhost:9333: get master localhost:9333 configuration: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp [::1]:19333: connect: connection refused"

**Comment by chrislusf** [comment]
https://github.com/seaweedfs/seaweedfs/wiki/FAQ#grpc-ports
